### PR TITLE
Remove CCAControlTypeFilters from Axe.Windows.Rules

### PR DESCRIPTION
**Pull request by RobGallo**
#### Describe the change

The class CCAControlTypeFilters was unnecessary because the integer values of the UIA control types could more directly be enumerated by the ControlType class. In addition, CCAControlTypeFilters was removed because it was made part of the Rules assembly, whose responsibility is only to run automated rules -- the Rules assembly should never know anything about the color contrast analyzer, or report any type information other than that provided by the rules.

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

**Comment by DaveTryon** [comment]
>             if (this.Dic.ContainsKey(id))

(aside): this is a perfect place for a TryGetValue

---
Refers to: src/AccessibilityInsights.Core/Types/TypeBase.cs:91 in 43e81b9. [](commit_id = 43e81b97df67229ad5e7c62435737c438a754350, deletion_comment = False)

File: src/AccessibilityInsights.Core/Types/TypeBase.cs [hunk 1]
```diff
@@ -115,5 +115,10 @@ public bool Exists(int id)
         {
             return this.Dic.ContainsKey(id);
         }
+
+        /// <summary>
+        /// the values of the types contained in the inheriting class
```
Comment (DaveTryon) [inline src/AccessibilityInsights.Core/Types/TypeBase.cs]: I'd omit "values of the ", like this:

```suggestion
        /// the types contained in the inheriting class
```

File: src/AccessibilityInsights.Core/Types/TypeBase.cs [hunk 2]
```diff
@@ -115,5 +115,10 @@ public bool Exists(int id)
...
+        /// </summary>
+        public IEnumerable<int> Values => Dic.Keys;
```
Comment (DaveTryon) [inline src/AccessibilityInsights.Core/Types/TypeBase.cs]: Should this be called Types?

```suggestion
        public IEnumerable<int> Types => Dic.Keys;
```

Comment (RobGallo) [reply src/AccessibilityInsights.Core/Types/TypeBase.cs]: I don't think so. I consider this to be the values of the types.

**Pull request merged**